app/src/main/res/values/config-common.xml

```diff
@@ -28,8 +28,7 @@
     <bool name="config_default_vibration_enabled">false</bool>
     <integer name="config_max_vibration_duration">100</integer>
 
-    <integer name="config_default_longpress_key_timeout">300</integer>
-    <integer name="config_max_longpress_timeout">700</integer>
+    <integer name="config_max_longpress_timeout">1500</integer>
```

Comment:
Why did you increase it so much? As far as I know, there was no request for longer times in OpenBoard or HeliBoard, so I don't think there is any need.
This makes precise setting (10 ms steps) more difficult.

Reply:
oh, sorry... I read the changes before the actual PR text